GeorgieGirl24/rails_engine
# Most revenue

closes #24 
Tests:
  * Add Merchant Model test: for .most_revenue to return the correct number of Merchants
  * Add test to check that the method .most_revenue could return the correct Merchants

MVC:
  * Add Api::V1::Merchants::StatisticsController #most_revenue to Serialize the correct number of Merchants.
